README.md

```diff
@@ -15,6 +15,47 @@ projects.
 For information on how to contribute, consult [the contributing
 guidelines](./CONTRIBUTING.md).
 
+## Project status
+
+Components shipped from this repository are at different maturity levels. The
+status for each component is mentioned in their respective Readme files. The
+component status falls under one of the following categories:
+
+### Development
+
+Component is currently in development and is not available on
+[Nuget](https://www.nuget.org/).
+
+### Alpha
+
+The component is ready to be used for limited non-critical workloads and the
+authors of this component would welcome your feedback. Bugs and performance
+problems should be reported, but component owners might not work on them right
+away. Components can go through significant breaking changes and there are no
+backward compatibility guarantees. Package in this status is appended by
+`-alpha` extension.
+
+### Beta
+
+Same as Alpha, but comparatively more stable. Package in this status is appended
+by `-beta` extension.
+
+### RC
+
+Component is close to stability. There might be minimal breaking changes between
+releases. A component at this stage is expected to have had exposure to
+non-critical production workloads already during its **Alpha/Beta** phase,
+making it suitable for broader usage. Package in this status is appended by
+`-rc` extension.
+
+### Stable
+
+The component is ready for general availability. Bugs and performance problems
+should be reported and there's an expectation that the component owners will
+work on them. Breaking changes, including configuration options and the
```

Comment:
simply state, the component is expected to follow semver20

Reply:
updated